ocean_lib/test/test_config.py

```diff
@@ -0,0 +1,63 @@
+#
```

Comment:
I think there is already a test_config.py. Also, ocean_lib/test/ is for integration tests, not for unit tests. Can we use the other test_config.py instead?

Reply:
Are you referring to `test_config_provider.py`? I don't see the "other `test_config.py`".

I think that `ocean.py/tests/integration` is the appropriate place to put integration tests and that `ocean_lib/test/test_config.py` is correctly following the convention for unit tests (to place them in a `test` folder that is a sibling of the file under test).

Reply:
ok, works for me, thanks!